Rg.Plugins.Popup/Platforms/Ios/Extensions/PlatformExtension.cs

```diff
@@ -97,5 +97,20 @@ public static void UpdateSize(this PopupPageRenderer renderer)
             else if (needForceLayout)
                 currentElement.ForceLayout();
         }
+
+        public static UIWindow GetKeyWindow(this UIApplication application)
+        {
+            if (!UIDevice.CurrentDevice.CheckSystemVersion(13, 0))
+                return UIApplication.SharedApplication.KeyWindow;
+
+            var window = application
```

Comment:
i noticed in the add there is mention of

```csharp
 if (UIApplication.SharedApplication.ConnectedScenes.ToArray()
                    .FirstOrDefault(x => x.ActivationState == UISceneActivationState.ForegroundActive && x is UIWindowScene) is UIWindowScene connectedScene)
```

your version here thanks to @xleon seems to be much cleaner than this. Just tagging this for a future consideration.

Reply:
What I´m not sure is if we need this part (which my code lacks): 
`x.ActivationState == UISceneActivationState.ForegroundActive`. 
What I understood from the SO thread is that we don't. But I need to test it.